### ISSUE TITLE
when set pos_weights for multilabel-classification,it can not train on multi-gpus

### ISSUE THREAD

**Issue by MrRace** [ThilinaRajapakse/simpletransformers#665]
**Describe the bug**
For `multilabel_classification.py` the `pos_weights` does not support training on multi-gpus.

**To Reproduce**
Use the example from `simpletransformers\examples\text_classification\multilabel_classification.py`. When assign my weights of classes to  `pos_weight` . I set  `n_gpu=3`. When train with multi-gpus it comes the error：
```
Expected all tensors to be on the same device, but found at least two devices, cuda:0 and cuda:1
```

When do not pass my own pos_weights to parameter `pos_weight`, it can train on multi-gpus normally.

When pass my own pos_weights to parameter `pos_weight` and set it  `n_gpu=1` it can train  normally, but just use single GPU.

**Screenshots**
![image](https://user-images.githubusercontent.com/10300313/90873390-b1911700-e3d0-11ea-8c9e-ad5f9c0ea5da.png)

**environment (please complete the following information):**
 - Linux
- Python 3.7

**Comment by elulue** [comment]
did u try distributed data parallel?

**Comment by MrRace** [comment]
> did u try distributed data parallel?

Yeah, I use multi-gpus

**Comment by washcycle** [comment]
This is related to the unresolved and closed issue #228.
